### ISSUE TITLE
#stock_candle not clear

### ISSUE THREAD

**Issue by AndreaGVN** [Finnhub-Stock-API/finnhub-python#52]
I admit I am not a sharp python user, but could someone elucidate me what the bold text in the following line of code stands for ? Time range ? Idk. 
Code : 
`res = finnhub_client.stock_candles('AAPL', 'D', **1590988249**, **1591852249**)`

I honestly have no idea how to tackle this, if someone could be so kind to explain how to deal with it would be great.

**Comment by onnnnn** [comment]
Yes, I think you are right. It's time range and both in *Timestamp* format.
You may reference [this article](https://flexiple.com/python/python-timestamp/) to have a clear understanding on Python *Timestamp*.

Since `stock_candles()` takes timestamp integer as parameter, you may pass `int(time.time())` as current timestamp with python `time` module.
